.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,8 +6,8 @@
      commits if your PR already received reviews or comments.
```

Comment:
This should not be here at all, I'll clean it up before moving this out of draft.